web/frontend/src/components/slam.vue

```diff
@@ -95,6 +104,77 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
   });
 };
 
+const executeMoveOnMap = async () => {
+  moveClick = false;
+
+  /*
+   * set request name
+   * here we set the name of the motion service the user is using
+   */
+  motionServiceReq.setName('builtin');
+
+  const value = await fetchSLAMPose(props.name);
+  // set pose in frame
+  const destination = new commonApi.Pose();
+  destination.setX(Math.abs(destinationMarker.x - value.getX()));
+  destination.setY(Math.abs(destinationMarker.z - value.getY()));
+  destination.setZ(value.getZ());
```

Comment:
I believe the pose should be specified in the XZ plane, so we should be setting the destination Y to `value.getY()` and the destination Z to the new Z coordinate.

Reply:
I agree since I believe viam is a z-forward coordinate system and thats currently how we return pose from carto (although we can always change this to abide by the viam coordinate system should I differ). Once this has been confirmed, lets also make sure the axes compass is updated to match (currently it shows XY)